Description: link libxml2.la with the python module explictly
Author: Aron Xu <aron@debian.org>
Last-Update: 2024-03-18
Bug-Debian: https://bugs.debian.org/1059094

Index: libxml2/python/Makefile.am
===================================================================
--- libxml2.orig/python/Makefile.am
+++ libxml2/python/Makefile.am
@@ -22,7 +22,8 @@ pyexec_LTLIBRARIES = libxml2mod.la
 
 libxml2mod_la_SOURCES = libxml.c libxml_wrap.h types.c
 nodist_libxml2mod_la_SOURCES = libxml2-py.h libxml2-py.c
-libxml2mod_la_LDFLAGS = $(AM_LDFLAGS) $(PYTHON_LDFLAGS) -module -avoid-version
+libxml2mod_la_LDFLAGS = $(AM_LDFLAGS) $(PYTHON_LDFLAGS) -module -avoid-version \
+	$(top_builddir)/libxml2.la
 libxml2mod_la_LIBADD = $(top_builddir)/libxml2.la $(PYTHON_LIBS)
 
 BUILT_SOURCES = libxml2-export.c libxml2-py.h libxml2-py.c
